common/lib/xmodule/xmodule/static_content.py

```diff
@@ -122,9 +122,11 @@ def _write_js(output_root, classes):
     js_fragments = set()
     for class_ in classes:
         module_js = class_.get_javascript()
+        # It will enforce 000 prefix for xmodule.js.
+        js_fragments.add((0, 'js', module_js.get('xmodule_js')))
```

Comment:
Did you look into doing this by stating dependencies instead of assuming the existence of xmodule_js.js?